notebooks/monte_carlo_case_study_nigeria_data.ipynb

```diff
@@ -0,0 +1,549 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.barplot(src_beta_coefficients)\n",
+    "plt.title(\"Beta coefficients of the parameters\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is labelling for the x-axis of the plots with regards to the selected parameters\n",
+    "\n",
+    "0: 'loads_t.p_set <br>\n",
+    "1: 'generators_t.p_max_pu.loc[:, n.generators.carrier == \"solar\"]'<br>\n",
+    "2: 'generators_t.p_max_pu.loc[:, n.generators.carrier == \"onwind\"]'<br>\n",
+    "3: 'generators.capital_cost.loc[n.generators.carrier == \"solar\"]'<br>\n",
+    "4: 'generators.capital_cost.loc[n.generators.carrier == \"onwind\"]'<br>\n",
+    "5: 'generators.weight.loc[n.generators.carrier == \"onwind\"]'<br>\n",
+    "6: 'generators.weight.loc[n.generators.carrier == \"solar\"]"
```

Comment:
The plots labels must be fixed on the plot, otherwise it's a bit difficult to comprehend.

Reply:
We might have a much larger plot if the label is inserted directly into the x-axis. I tried using labels but it took a considerably large amount of space in the plot, covering some of the bars in the plot.

I am open to different ideas.

Reply:
Fair point. Can we use short yet meaningful version of the labels?